src/app/models/PlanModel.ts

```diff
@@ -0,0 +1,40 @@
+import * as mongodb from 'mongodb';
+
+class PlanModel {
+  _id: mongodb.ObjectId;
+  name: string;
+  description: string;
+  thumbnail: string;
+  tags: string[];
+  likes: number;
+  classes: string[];
+  createdBy: string;
+  visible: boolean;
+  createdAt = Date;
+
+  constructor(
+    name = '',
+    description = '',
+    thumbnail = '',
+    tags = [],
+    _likes = 0,
+    classes = [],
+    createdBy = '',
+    visible = false,
+    createdAt = Date,
```

Comment:
```suggestion
    createdAt = new Date(),
```

Reply:
@jpatrici I made this change, but I am getting this error: "Type 'Date' is missing the following properties from type 'DateConstructor': prototype, parse, UTC, now"

Reply:
but this is not the way to create a Date. What you need to do here is giving a default value to `createdAt`, and it should be `new Date()`. `Date` is the type

Reply:
I had a syntax error earlier in the file. Fixed!